src/multitask_personalization/envs/pybullet/pybullet_env.py

```diff
@@ -108,41 +102,8 @@ def __init__(
         )
 
         # Create human.
-        human_creation = HumanCreation(
-            self.physics_client_id, np_random=self._rng, cloth=False
-        )
-        self.human = Human([], controllable=False)
-        self.human.init(
-            human_creation,
-            static_human_base=True,
-            impairment="none",
-            gender="male",
-            config=None,
-            id=self.physics_client_id,
-            np_random=self._rng,
-        )
-        set_pose(
-            self.human.body, self.task_spec.human_base_pose, self.physics_client_id
-        )
-
-        # Use some default joint positions from assistive gym first.
-        joints_positions = [
-            (self.human.j_right_elbow, -90),
-            (self.human.j_left_elbow, -90),
-            (self.human.j_right_hip_x, -90),
-            (self.human.j_right_knee, 80),
-            (self.human.j_left_hip_x, -90),
-            (self.human.j_left_knee, 80),
-            (self.human.j_head_x, 0.0),
-            (self.human.j_head_y, 0.0),
-            (self.human.j_head_z, 0.0),
-        ]
-        self.human.setup_joints(
-            joints_positions, use_static_joints=True, reactive_force=None
-        )
-        # Now set arm joints using task spec.
-        self.human.set_joint_angles(
-            self.human.right_arm_joints, self.task_spec.human_joints
+        self.human = create_human_from_spec(
```

Comment:
Would this not result in two `Human` objects in the env, since `ROMModel` in `hidden_spec` also created an instance of `Human`?

Reply:
Got it!